GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
```

Comment:
But you don't need that. It's not the work of the parser to sanitize the color name. It's the work of the template filters, like [here](https://github.com/AliSoftware/SwiftGen/blob/master/templates/colors-default.stencil#L23).

That's because people could want to customize their template and use the real name (with spaces and all) for purposes other than just use them as the swift identifier, but to use them to generate comments for each color, or to associate a color name as a String or implement a nice CustomStringConvertible or whatnot. But if they use it as a swift identifier, that's their responsibility to call the `swiftIdentifier` Stencil filter on the name in their template in the appropriate place(s).